pwd/pwd.py

```diff
@@ -0,0 +1,25 @@
+import ffilib
+import uctypes
+import ustruct as struct
+
+from ucollections import namedtuple
+
+
+libc = ffilib.libc()
+
+if libc:
```

Comment:
Other ffilib-using modules don't have this if, maybe it's not needed here either?

Reply:
I've used os as reference, but will remove

Reply:
I see. Well, I believe there were reports that libc can't be imported e.g. on Windows, maybe on MacOS. So, that "if" was added to not make the module fail on import and just offer constants and things from "uos" module. We don't need such handling in most other modules, so thanks for removing it.